content/influxdb/v1.8/about_the_project/releasenotes-changelog.md

```diff
@@ -9,7 +9,12 @@ menu:
 v2: /influxdb/v2.0/reference/release-notes/influxdb/
 ---
 
+## v1.8.6 [2021-05-21]
+
+This release is for InfluxDB Enterprise 1.8.6 customers only. No OSS-specific changes were made for InfluxDB 1.8.6--updates were made to the code base to support [InfluxDB Enterprise 1.8.6](/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog/#v186-2021-05-21).
```

Comment:
To be clear, is it true that folks only need to install this OSS release if they're using OSS instances in conjunction with their Enterprise instance? And there'd be no practical reason for OSS only folks to download this release.